ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/qos_profiles.schema.yml

```diff
@@ -111,6 +165,60 @@ keys:
                     description: |
                       Supported options are platform dependent
                       Example: "< rate > kbps", "1-100 percent", "< rate > pps"
+              random_detect:
+                type: dict
+                keys:
+                  ecn:
+                    description: Explicit Congestion Notification
+                    type: dict
+                    keys:
+                      count:
+                        type: bool
+                        description: |
+                          Enable counter for random-detect ECNs
+                      threshold:
+                        type: dict
+                        keys:
+                          units:
+                            required: true
+                            type: str
+                            valid_values: ["segments", "bytes", "kbytes", "mbytes"]
+                            description: |
+                              Units to be used for the threshold values.
+                              This should be one of segments, byte, kbytes, mbytes.
```

Comment:
Avoid repeating the valid values.
Also should this be singular `unit` since it we set only one "unit" to be used on the other values?

Reply:
Agreed